lib/changeNIDMtoTabDat.m

```diff
@@ -267,28 +272,36 @@
     FWHMUnits = strrep(strrep(strrep(strrep(strrep(get_value(searchSpace.('nidm_voxelUnits')), '\"', ''), '[', ''), ']', ''), ',', ''), '"', '');
     
     ftrTemp{rowCount, 1} = ['FWHM = %3.1f %3.1f %3.1f ', FWHMUnits '; %3.1f %3.1f %3.1f {voxels}'];
-    
-    %Store the FWHM in units and voxels
-    unitsFWHM = str2num(get_value(searchLinkedToCoord.('nidm_noiseFWHMInUnits')));
-    voxelsFWHM = str2num(get_value(searchLinkedToCoord.('nidm_noiseFWHMInVoxels')));
-    
-    ftrTemp{rowCount,2} = [unitsFWHM, voxelsFWHM];
+
+    if parametric
+        %Store the FWHM in units and voxels
+        unitsFWHM = str2num(get_value(searchLinkedToCoord.(context('nidm_noiseFWHMInUnits'))));
+        voxelsFWHM = str2num(get_value(searchLinkedToCoord.(context('nidm_noiseFWHMInVoxels'))));
+        
+        ftrTemp{rowCount,2} = [unitsFWHM, voxelsFWHM];
+    else
+        ftrTemp{rowCount,2} = [NaN, NaN];
+    end
     
     %Volume
     
     rowCount = rowCount+1;
     
-    volumeUnits = get_value(searchLinkedToCoord.('nidm_searchVolumeInUnits'));
+    volumeUnits = get_value(searchLinkedToCoord.(context('nidm_searchVolumeInUnits')));
     if(ischar(volumeUnits))
         volumeUnits = str2double(volumeUnits);
     end
     
-    volumeResels = get_value(searchLinkedToCoord.('nidm_searchVolumeInResels'));
-    if(ischar(volumeResels))
-        volumeResels = str2double(volumeResels);
+    if parametric
+        volumeResels = get_value(searchLinkedToCoord.(context('nidm_searchVolumeInResels')));
+        if(ischar(volumeResels))
+            volumeResels = str2double(volumeResels);
+        end
+    else
+        volumeResels = NaN;
     end
```

Comment:
Does SnPM not have any resel statistics?

Reply:
Non-parametric FWE does not use random field theory which means that there is no need to compute resels. I've added a comment.